src/domain/user/user_entity.go

```diff
@@ -20,7 +21,7 @@ const (
 	userCanProposalSkillLength = 5
 )
 
-func NewUser(id UserID, name UserName, email Email, password Password, profile string, skill []Skill) (*User, error) {
+func NewUser(id UserID, name UserName, email Email, password Password, profile string, skill []Skill, resume []Resume) (*User, error) {
```

Comment:
```suggestion
func NewUser(id UserID, name UserName, email Email, password Password, profile string, skills []Skill, resumes []Resume) (*User, error) {
```
細かいですがスライスは複数形にしておくと変数名として扱いやすいです！

Reply:
ご指摘ありがとうございいます。
確かにそうですね...。
修正して、マージします！